### DISCUSSION TITLE
Make borders for query builder items consistent

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5793] [Feature Requests]
![image](https://user-images.githubusercontent.com/37256050/236002760-56ba50d1-23b0-4aad-9ff5-1350b3b948d4.png)

To solve this, just need to make the "True" list have the same border color as the "Current" list. Minimal change.

**Comment by maxpatiiuk** [comment]
or make "Current" list border less dark to match "True" list (as current border is a bit too dark - personal opinion)

**Comment by maxpatiiuk** [comment]
@chanulee1 Can you take a look at this?

You can use React DevTools to locate where in the code a given component is defined and then in the resulting file search for the [tailwind style](https://tailwindcss.com/docs/background-color) responsible for background

I would be happy to meet with you about this and fix it together

**Comment by grantfitzsimmons** [comment]
@carlosmbe See @maxpatiiuk's comment on this! This should be a small fix :smile:

**Comment by emenslin** [comment]
Can recreate in edge (7.9.6)
